services/src/main/java/com/linkedin/kmf/services/ConsumeService.java

```diff
@@ -196,43 +202,72 @@ public boolean isRunning() {
   }
 
   private class ConsumeMetrics {
+    public final Metrics metrics;
     private final Sensor _bytesConsumed;
     private final Sensor _consumeError;
     private final Sensor _recordsConsumed;
     private final Sensor _recordsDuplicated;
     private final Sensor _recordsLost;
     private final Sensor _recordsDelay;
+    private final Sensor _recordsDelayed;
+
+    public ConsumeMetrics(Metrics metrics, final Map<String, String> tags) {
+      this.metrics = metrics;
 
-    public ConsumeMetrics(Metrics metrics, Map<String, String> tags) {
       _bytesConsumed = metrics.sensor("bytes-consumed");
-      _bytesConsumed.add(new MetricName("bytes-consumed-rate", METRIC_GROUP_NAME, tags), new Rate());
+      _bytesConsumed.add(new MetricName("bytes-consumed-rate", METRIC_GROUP_NAME, "The average number of bytes per second that are consumed", tags), new Rate());
```

Comment:
I have considered this. But `metrics.metricName()` is not available in kafka 0.9.0.1. We can replace all usage of deprecated Kafka APIs the next time we upgrade Kafka Monitor's dependency on Kafka.